### PR TITLE
[DocumentIntelligence] Documentations + Samples updates

### PR DESCRIPTION
**Pull request by mssfang**
Update

-  [X] CHANGELOG
-  [x] README
-  [x] Samples: addons, markdown

### CODE REVIEW

File: sdk/documentintelligence/azure-ai-documentintelligence/CHANGELOG.md [hunk 1]
```diff
@@ -12,16 +12,93 @@
 
 ## 1.0.0-beta.1 (2023-11-16)
 
-Version 1.0.0-beta.1 is a preview of our efforts in creating an Azure AI Document Intelligence client library that is developer-friendly
-and idiomatic to the Java ecosystem. The principles that guide
-our efforts can be found in the [Azure SDK Design Guidelines for Java](https://azure.github.io/azure-sdk/java_introduction.html).
+_**Note: [Form Recognizer has been rebranded to Document Intelligence](https://mixedrealitywiki.com/display/VTI/Document+Intelligence+2023-10-31-preview#DocumentIntelligence20231031preview-Rebranding)**_
 
+This marks the first preview of `azure-ai-documentintelligence` REST level client library for the `Azure AI Document
```
Comment (YalinLi0312) [inline sdk/documentintelligence/azure-ai-documentintelligence/CHANGELOG.md:17]: Rest level client is just for JS.

File: sdk/documentintelligence/azure-ai-documentintelligence/README.md [hunk 2]
```diff
@@ -32,6 +32,25 @@ It includes the following main features:
...
 
+> Note: This version of the client library defaults to the `"2023-10-31-preview"` version of the service.
+
+This table shows the relationship between SDK versions and supported API versions of the service:
+
+| SDK version  |Supported API version of service
+|--------------|-
+| 1.0.0-beta.1 | 2023-10-31-preview
+
+> Note: `Azure AI Document Intelligence` service (formerly known as Form Recognizer), start targeting service API
```
Comment (YalinLi0312) [inline sdk/documentintelligence/azure-ai-documentintelligence/README.md:43]: nit: looks this line is duplicate with line 35

File: sdk/documentintelligence/azure-ai-documentintelligence/CHANGELOG.md [hunk 3]
```diff
@@ -12,16 +12,94 @@
 
 ## 1.0.0-beta.1 (2023-11-16)
 
-Version 1.0.0-beta.1 is a preview of our efforts in creating an Azure AI Document Intelligence client library that is developer-friendly
-and idiomatic to the Java ecosystem. The principles that guide
-our efforts can be found in the [Azure SDK Design Guidelines for Java](https://azure.github.io/azure-sdk/java_introduction.html).
+_**Note: [Form Recognizer has been rebranded to Document Intelligence](https://mixedrealitywiki.com/display/VTI/Document+Intelligence+2023-10-31-preview#DocumentIntelligence20231031preview-Rebranding)**_
```
Comment (HarshaNalluru) [inline sdk/documentintelligence/azure-ai-documentintelligence/CHANGELOG.md:15]: this link won't be accessible to customers as far as I understand, I think we should omit this

File: sdk/documentintelligence/azure-ai-documentintelligence/CHANGELOG.md [hunk 4]
```diff
@@ -12,16 +12,94 @@
...
-- Two client design:
-    - `DocumentIntelligenceClient` to analyze fields/values on custom documents, receipts, and document layout
-    - `DocumentIntelligenceAdministrationClient` to build custom models, manage the custom models on your account and build classifiers
-- Authentication with API key supported using `AzureKeyCredential("<api_key>")` from `com.azure.core.credential`
-- Reactive streams support using [Project Reactor](https://projectreactor.io/).
+### Features Added
+
+- [azure-ai-documentintelligence](https://github.com/Azure/azure-sdk-for-java/tree/main/sdk/documentintelligence/azure-ai-documentintelligence) 
+  is the new package, replacing [azure-ai-formrecognizer](https://github.com/Azure/azure-sdk-for-java/tree/main/sdk/formrecognizer/azure-ai-formrecognizer)
+  package. The new package supports a REST level client, which is part of the new generation of Azure SDKs to simplify
```
Comment (HarshaNalluru) [inline sdk/documentintelligence/azure-ai-documentintelligence/CHANGELOG.md:29]: REST level client is specific to JS

File: sdk/documentintelligence/azure-ai-documentintelligence/CHANGELOG.md [hunk 5]
```diff
@@ -12,16 +12,94 @@
...
+  when compared to the API versions supported by the `azure-ai-formrecognizer` library.
+  - [Markdown content format](https://mixedrealitywiki.com/display/VTI/Document+Intelligence+2023-10-31-preview#DocumentIntelligence20231031preview-MarkdownConversion)
+  
+    Supports output with Markdown content format along with the default plain _text_. For now, this is only supported for 
+    "prebuilt-layout". Markdown content format is deemed a more friendly format for LLM consumption in a chat or 
+    automation use scenario. Service follows the GFM spec ([GitHub Flavored Markdown](https://github.github.com/gfm/))
+    for the Markdown format. Also introduces a new _contentFormat_ property with value "text" or "markdown" to indicate 
+    the result content format.
+
+  - [Query Fields](https://mixedrealitywiki.com/display/VTI/Document+Intelligence+2023-10-31-preview#DocumentIntelligence20231031preview-QueryFields)
```
Comment (HarshaNalluru) [inline sdk/documentintelligence/azure-ai-documentintelligence/CHANGELOG.md:43]: exclude link?

File: sdk/documentintelligence/azure-ai-documentintelligence/CHANGELOG.md [hunk 6]
```diff
@@ -12,16 +12,94 @@
...
+    "type": "currency",
+    "valueCurrency": {
+      "amount": 123.45,
+      "currencySymbol": "$",
+      "currencyCode": "USD"
+    }
+  }
+  ...
+  ```
+- API shapes have been designed from scratch to support the new Rest level client for the Document Intelligence service.
```
Comment (HarshaNalluru) [inline sdk/documentintelligence/azure-ai-documentintelligence/CHANGELOG.md:93]: Rest level client, specific to JS

File: sdk/documentintelligence/azure-ai-documentintelligence/CHANGELOG.md [hunk 7]
```diff
@@ -12,16 +12,94 @@
...
+  Please refer to the [README](https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/documentintelligence/azure-ai-documentintelligence/README.md)
+  and [Samples](https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/documentintelligence/azure-ai-documentintelligence/src/samples/README.md) 
+  for more understanding.
+
+### Other Changes
+
+#### Retirements/Deprecations
+
+- `"prebuilt-businessCard"` model is retired.
+- `"prebuilt-document"` model is retired, this model is essentially `"prebuilt-layout"` with `features=keyValuePairs` specified. _(This is only supported as an optional feature for "prebuilt-layout" and "prebuilt-invoice".)_
```
Comment (HarshaNalluru) [inline sdk/documentintelligence/azure-ai-documentintelligence/CHANGELOG.md:103]: `features=keyValuePairs`

Update this to how the option looks like in a JAVA code snippet

**Review by HarshaNalluru — APPROVED** [review]

**Pull request merged**